ModelBundle/Repository/ContentRepository.php

```diff
@@ -374,25 +374,40 @@ public function findPaginatedLastVersionByContentTypeAndSite($contentType = null
      * @param int|null            $siteId
      *
      * @return int
+     *
+     * @deprecated will be removed in 1.3.0, use countByContentTypeAndSiteInLastVersionWithFilter
      */
     public function countByContentTypeInLastVersionWithFilter($contentType = null, FinderConfiguration $configuration = null, $siteId = null)
     {
+        return $this->countByContentTypeAndSiteInLastVersionWithFilter($contentType, $configuration, $siteId);
+    }
+
+    /**
+     * {@inheritDoc}
+     * @see \OpenOrchestra\ModelInterface\Repository\ContentRepositoryInterface::countByContentTypeAndSiteInLastVersionWithFilter()
+     */
+    public function countByContentTypeAndSiteInLastVersionWithFilter(
+        $contentType = null,
+        FinderConfiguration $configuration = null,
+        $siteId = null
+    ) {
         $qa = $this->createAggregateQueryWithContentTypeFilter($contentType);
         $qa = $this->generateFilter($qa, $configuration);
         $qa->match($this->generateDeletedFilter());
         if (!is_null($siteId)) {
             $qa->match($this->generateSiteIdAndNotLinkedFilter($siteId));
         }
-        $elementName = 'content';
-        $this->generateLastVersionFilter($qa, $elementName);
+        $this->generateLastVersionFilter($qa, 'content');
 
-        return $this->countDocumentAggregateQuery($qa, $elementName);
+        return $this->countDocumentAggregateQuery($qa);
     }
 
     /**
      * @param string|null $contentType
      *
      * @return int
+     *
+     * @deprecated will be removed in 1.3.0, use countByContentTypeAndSiteInLastVersion
```

Comment:
can you add a trigger error in deprecated method for instance
`
 @trigger_error('The '.__METHOD__.' method is deprecated since version 1.2.0 and will be removed in 1.3.0', E_USER_DEPRECATED);`